File: programming languages/31. prova esame/Prenotazione.java
class Prenotazione {
    private final int codiceCampo;
    private final int oraInizio;

    public Prenotazione(int codiceCampo, int oraInizio) {
        this.codiceCampo = codiceCampo;
        this.oraInizio = oraInizio;
    }

    public int getCodice() {
        return codiceCampo;
    }

    @Override
    public String toString() {
        return "(" + codiceCampo + ", " + oraInizio + ")";
    }
}

File: Fondamenti_programmazione_II/Python/Esami/Gestione_circolo_sportivo/Prenotazione.py
class Prenotazione:
    def __init__(self, campo, ora):
        self._campo = campo
        self._ora = ora
    
    def getOra(self):
        return self._ora

    def getCodice(self):
        return self._campo

    def __str__(self):
        return "(" + str(self._campo) + "," + str(self._ora) + ")"